base-theme/assets/css/variables.scss

```diff
@@ -3,15 +3,15 @@ $max-content-width: 1447px;
 $desktop-header-height: 116px;
 $desktop-header-max-width: 1664px;
 $extra-small-screen-width: 400px;
-$enable-responsive-font-sizes: true;
-$font-size-base: 0.8rem;
-$headings-font-weight: 600;
-$table-cell-padding: 0.5rem;
-$offcanvas-width: 85vw;
-$transparent: #ffffff00;
+$enable-responsive-font-sizes: true; /* not found */
```

Comment:
@gumaerc I was unable to find any usage of these variables and a few other classes pointed below, are these being/will be used somewhere or we can remove them?

Reply:
I think we can safely delete any unused variables - if we need them later, they'll be in the git history 🙂

Reply:
@alicewriteswrongs If you get time, can you also review this PR?